core/includes/database/charset_converter.inc

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
```

Comment:
I don't think `@file` is necessary if we only have one class in the file, as is the case here. They'd just be redundant with each other.